SmartDeviceLinkTests/DevAPISpecs/SDLFileManagerSpec.m

```diff
@@ -172,6 +172,22 @@ @implementation FileManagerSpecHelper
             });
         });
 
+        describe(@"after receiving a ListFiles error with a resultCode ENCRYPTION_NEEDED", ^{
+            beforeEach(^{
+                SDLListFilesOperation *operation = testFileManager.pendingTransactions.firstObject;
+                NSMutableDictionary *userInfo = [[NSError sdl_fileManager_unableToStartError].userInfo mutableCopy];
+                userInfo[@"resultCode"] = SDLResultEncryptionNeeded;
+                NSError *errorWithResultCode = [NSError errorWithDomain:[NSError sdl_fileManager_unableToStartError].domain code:[NSError sdl_fileManager_unableToStartError].code userInfo:userInfo];
```

Comment:
Replace these lines with just creating the error on one line. Don't copy a bunch of stuff and modify the error.

Reply:
I refactored the tests that check for an error response from the `ListFiles` request so it is easier to read.